### PR TITLE
fix(drop-down): use dom-align instead of metal-position

### PR DESCRIPTION
**Pull request by bryceosterhaus**
fixes https://github.com/liferay/clay/issues/2722

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -15046,7 +15051,7 @@ metal-keyboard-focus@^2.0.0:
     metal "^2.0.0"
     metal-events "^2.2.3"
 
-metal-position@^2.1.0, metal-position@^2.1.2:
+metal-position@^2.1.0:
```
Comment (bryceosterhaus) [inline yarn.lock]: note that metal-position is no longer a dependency for clay v3, but it is still used on clayui.com via tooltip v2

**Comment by bryceosterhaus** [comment]
cc @thektan check out the CSB link above `fragrant-rgb-dlskl` to verify your issue is fixed

**Comment by wincent** [comment]
We might want to create a task to move off of metal-position in liferay-portal too, so that we don't have two very similar dependencies floating around in there (for example, in frontend-js-tooltip-support-web and possibly elsewhere).

**Comment by thektan** [comment]
@bryceosterhaus Thanks for the fix! Looks good! 👌

**Comment by bryceosterhaus** [comment]
@wincent whats the best way for us to additionally make this change in portal? Should I create a sibling issue somewhere, or should I just go ahead and make the change for portal? Thinking for future fixes like this in the future as well.

**Comment by wincent** [comment]
@bryceosterhaus I don't think the changes need to go in atomically as a single change so it should be fine to create a sibling issue in Jira. Probably an IFI? If the change is trivial I would just do it; in this case it doesn't seem like it would be trivial though, so we can just create the IFI and then with the help of @esanzp and @jbalsas figure out the priority of it.

**Comment by bryceosterhaus** [comment]
@matuzalemsteles good catch on those issues. I just added a fix for both, let me know if you see any issues.

**Comment by wincent** [comment]
I created https://issues.liferay.com/browse/IFI-1234 to track the migration away from metal-position inside liferay-portal itself.

**Comment by bryceosterhaus** [comment]
oh my bad, I created an issue but forgot to link it here. https://issues.liferay.com/browse/IFI-1226

I'll close my issue and just link to yours

**Comment by wincent** [comment]
And my JSSR (Jira Search Success Rate) is close to 0% so I didn't even try searching for a ticket before creating mine. 😂

**Pull request merged**